### ISSUE TITLE
Disable Do Not Track and remove option in privacy settings

### ISSUE THREAD

**Issue by orazioedoardo** [GrapheneOS/Vanadium#381]
It's useless and aids fingerprinting.

**Comment by orazioedoardo** [comment]
Umh, ok... thought it made sense. Safari did just that back in 2019.

**Comment by thestinger** [comment]
It's enabled by default in Vanadium so as long as no one disables it, it's always the same. We could force enable it but there are plenty of other settings people could change with a similar impact.

**Comment by orazioedoardo** [comment]
> It's enabled by default in Vanadium so as long as no one disables it

Yeah, but doesn't having it enabled make easier to distinguish Vanadium from Chromium?

**Comment by thestinger** [comment]
People want us to add a bunch of features making it easier to distinguish from Chromium.
